src/Tribe/Views/V2/Assets.php

```diff
@@ -577,12 +578,12 @@ public function should_enqueue_frontend() {
 		 */
 		if (
 			defined( 'ELEMENTOR_PATH' )
-
 			&& ! empty( ELEMENTOR_PATH )
-
-			&& isset( $_GET[ 'elementor-preview' ] )
+			&& isset( $_GET['elementor-preview'] ) // phpcs:ignore WordPress.Security.NonceVerification.Recommended
 		) {
-			return $this->should_enqueue = true;
+			$this->should_enqueue_frontend = true;
```

Comment:
There is no `$this->should_enqueue` I'm assuming it was meant to go into `$this->should_enqueue_frontend` as putting it in `$should_enqueue` (local) would be meaningless with the immediate return.